Source/WebCore/svg/SVGRadialGradientElement.cpp

```diff
@@ -98,6 +99,7 @@ void SVGRadialGradientElement::svgAttributeChanged(const QualifiedName& attrName
     if (PropertyRegistry::isKnownAttribute(attrName)) {
         InstanceInvalidationGuard guard(*this);
         updateRelativeLengthsInformation();
+        invalidateGradientResource();
         updateSVGRendererForElementChange();
```

Comment:
I think this call to updateSVGRendererForElementChange is not needed (I commented before, not sure if it was lost).

Reply:
Good catch!